jdaviz/configs/imviz/plugins/footprints/footprints.py

```diff
@@ -285,7 +294,62 @@ def _on_select_footprint_overlay(self, data):
         closest_overlay_label, closest_point = find_closest_polygon_point(
             click_x, click_y, overlay_data)
         self.overlay_selected = closest_overlay_label
-        self._highlight_overlay(closest_overlay_label, viewers=self.viewer.selected_obj)
+
+    def _handle_skewer_click(self, data):
+        """Spherical (great-circle) selection: only selects if the click is INSIDE a footprint.
+        If multiple overlays contain the click, picks the one with smallest area.

```

Comment:
maybe this should also be in the tooltip?  I was wondering whether it would select multiple, closest, etc.

Reply:
The longer-term goal is to select all matches, once footprint multiselect is compatible with the Footprints plugin.

Reply:
ok, depending on the resolution of the conversation related to that... if this stays with the single-select at least for now, it might still make sense to describe in the tool description/tooltip and we can always modify that once multiselect is included.